src/ChurchCRM/Emails/BaseEmail.php

```diff
@@ -6,15 +6,15 @@
 use ChurchCRM\dto\SystemConfig;
 use ChurchCRM\dto\SystemURLs;
 use ChurchCRM\Service\SystemService;
-use Mustache_Engine;
-use Mustache_Loader_FilesystemLoader;
+use Twig\Loader\FilesystemLoader;
+use Twig\Environment;
 use PHPMailer\PHPMailer\PHPMailer;
 
 abstract class BaseEmail
 {
     /** @var PHPMailer */
     protected $mail;
-    protected \Mustache_Engine $mustache;
+    protected $twig;
```

Comment:
I think this property can be typed (as well as the one above this)

```suggestion
    protected Environment $twig;
```